src/js/reducers/user.js

```diff
@@ -1,30 +1,53 @@
+import {
+  USER_LOADING,
+  USER_EXPIRED,
+  USER_FOUND,
+  USER_SIGNED_OUT,
+  SILENT_RENEW_ERROR,
+  SESSION_TERMINATED,
+  CLEAR_USER_ERROR
+} from '../constants'
+
 const defaultUser = {
-  expired: false,
+  oidc: null,
+  isFetching: false,
   userError: false
 }
 
 export default (state = defaultUser, action) => {
   switch (action.type) {
-    case 'redux-oidc/SILENT_RENEW_ERROR':
+    case SESSION_TERMINATED:
```

Comment:
Are `SESSION_TERMINATED`, `USER_SIGNED_OUT`, and `USER_EXPIRED` every handled differently?